test/CMakeLists.txt

```diff
@@ -145,9 +143,7 @@ if (FMT_PEDANTIC)
     target_include_directories(
       noexception-test PRIVATE ${PROJECT_SOURCE_DIR}/include)
     target_compile_options(noexception-test PRIVATE -fno-exceptions)
-    if (FMT_PEDANTIC)
```

Comment:
Why is it removed?

Reply:
This 'if (FMT_PEDANTIC)' is inside another 'if (FMT_PEDANTIC)' block.